index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html> 
+<html>
+<head> 
+    <title>Hugo Padilla's Portfolio</title>
+    <meta charset="UTF-8">
+    <meta name="author" content="Hugo Padilla">
+</head>
+
+<body>
+    <h1>Hugo Padilla</h1>
+
+    <section id = "About">
+    <h2>About</h2>
+    <p>My name is Hugo Padilla, and I am a software developer.</p>
+</section>
+
+<section id = "Experience">
+    <h2>Experience</h2>
+    <ul>
+        <li>Computer Science at Texas A&M - SA</li>
+        <li>Intern at Runwei (2024)</li>
+        </ul>
+
+    </section>
+
+<section id = "Connect">
+ <h2>Connect</h2>
+ <ul>
+    <li><a href = https://github.com/hpadi02></a> https://github.com/hpadi02</li>
```

Comment:
They ware you are formatting your anchor tag. make sure to review that again
besides, `<a href="your link here">The element to be clicked on</a>`
eg: `<li><a href="https://github.com/hpadi02">gitHub</a></li>`